app/jobs/s3_west_delivery_job.rb

```diff
@@ -1,31 +1,11 @@
 # frozen_string_literal: true
 
-# Posts zips to S3, if needed.
-# Notify ResultsRecorderJob, if posted.
 # @see PreservationCatalog::Aws for how S3 credentials and bucket are configured
 # @note This name is slightly misleading, as this class solely deals with AWS US West 2 endpoint
-class S3WestDeliveryJob < ZipPartJobBase
+class S3WestDeliveryJob < AbstractDeliveryJob
```

Comment:
This is straight-up refactoring, so I'd prefer not to rename existing jobs as part of this PR.

Reply:
I disagree.  The existing names are confusing, and the renames will help.   Or is that because it makes it harder to grok the changes in github UI?

(p.s. that's something JCoyne would say)